src/main/java/net/neoforged/neoforge/client/entity/animation/json/JsonAnimator.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.client.entity.animation.json;
+
+import net.minecraft.client.model.HierarchicalModel;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.entity.AnimationState;
+
+/**
+ * Use this to animate your entity with animations loaded from JSON files. You can create these animations using the
+ * "Animation to JSON Converter" plugin from the Blockbench plugins store.
+ */
+public final class JsonAnimator {
```

Comment:
I honestly see no reason for this with my previous comments. Though also, this would keep a stale reference to the hierarchical model, unless we're keeping this on the model instance itself for some reason? This just seems weird as its a wrapper to call methods already available in the model itself.

Reply:
The idea is you store it on the model itself, and then use `animator.animate(state, animation, ageInTicks)` instead of `this.animate(state, animation, ageInTicks)`. It's much simpler to use than `this.animate(state, JsonAnimationLoader.INSTANCE.getAnimationOrThrow(animation), ageInTicks)`. You can check [the original description](https://modrinth.com/mod/json-entity-animation) on the library I ported this from for a full example.

Reply:
My thought was this: Which is simpler?

This:
```java
private static final ResourceLocation MY_ANIMATION = MyMod.id("my_entity/my_animation");

private final JsonAnimator animator = new JsonAnimator(this);

    animator.animate(entity.state, MY_ANIMATION, ageInTicks);
```

Or this:
```java
private static final AnimationHolder MY_ANIMATION = JsonAnimatorLoader.INSTANCE.getAnimation(MyMod.id("my_entity/my_animation"));

    this.animate(entity.state, MY_ANIMATION, ageInTicks);
```

(I am agnostic on this, the decision is yours)